FreeRTOS/FreeRTOS-Plus-TCP
# IPv6/Multi : (IPv6TestingBranch) Update folder structure

IPv6/Multi : IPv6TestingBranch Update folder structure

Description
-----------

Follow #466 to update folder structure in labs/ipv6_multi.
- Add source folder.
- move include, portable directories into source folder.
- move all source files into source folder.

Test Steps
-----------

Use following commands to make sure it can pass compilation.
- cmake -S test/unit-test -B test/unit-test/build
- make -C test/unit-test/build/ all

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
